src/pages/BusinessBankAccount/NewPage.js

```diff
@@ -0,0 +1,196 @@
+import _ from 'underscore';
+import React from 'react';
+import PropTypes from 'prop-types';
+import {View, Text, Image} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import MenuItem from '../../components/MenuItem';
+import {Paycheck, Bank, Lock} from '../../components/Icon/Expensicons';
+import styles from '../../styles/styles';
+import TextLink from '../../components/TextLink';
+import Button from '../../components/Button';
+import Icon from '../../components/Icon';
+import colors from '../../styles/colors';
+import Navigation from '../../libs/Navigation/Navigation';
+import Permissions from '../../libs/Permissions';
+import CONST from '../../CONST';
+import TextInputWithLabel from '../../components/TextInputWithLabel';
+import AddPlaidBankAccount from '../../components/AddPlaidBankAccount';
+import ONYXKEYS from '../../ONYXKEYS';
+import CheckboxWithLabel from '../../components/CheckboxWithLabel';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import compose from '../../libs/compose';
+import bankDetailsImage from '../../../assets/images/bank-details.png';
+
+const propTypes = {
+    /** List of betas */
+    betas: PropTypes.arrayOf(PropTypes.string).isRequired,
+
+    ...withLocalizePropTypes,
+};
+
+class BusinessBankAccountNewPage extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.toggleTerms = this.toggleTerms.bind(this);
+        this.addManualAccount = this.addManualAccount.bind(this);
+
+        this.state = {
+            addMethodSelection: undefined,
```

Comment:
I think a comment would be good here:

```
// One of CONST.BANK_ACCOUNT.ADD_METHOD
```

Or something that just makes it slightly clearer that this state variable isn't a function, which is what I thought it would be at first. Also NAB I'd use `choice` instead of `selection` because it's fewer syllables 🙃

Reply:
yeah, that's a good point. It does have an action verb and sounds like a function name.

Reply:
Changing it to `bankAccountAddMethod` to match the `CONST` I think maybe it's a bit clearer that way.